src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AmbientValueAttribute.cs

```diff
@@ -0,0 +1,180 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+
+
+
+/*
+ */
+
+namespace System.ComponentModel
+{
+    using System;
+    using System.ComponentModel;
+    using System.Diagnostics;
+    using System.Globalization;
+    using System.Runtime.InteropServices;
+    //using System.Runtime.Serialization.Formatters;
```

Comment:
It was not needed. There are a couple of other places where we have that. Cleaned them up.